internal/command/migrate_to_v2/migrate_to_v2.go

```diff
@@ -167,34 +184,107 @@ func NewV2PlatformMigrator(ctx context.Context, appName string) (V2PlatformMigra
 	if err != nil {
 		return nil, err
 	}
+	migrator.resolveProcessGroups(ctx)
 	return migrator, nil
 }
 
-func (m *v2PlatformMigrator) Migrate(ctx context.Context) error {
+func (m *v2PlatformMigrator) rollback(ctx context.Context, tb *render.TextBlock) error {
 	var err error
 
-	tb := render.NewTextBlock(ctx, fmt.Sprintf("Migrating %s to the V2 platform", m.appCompact.Name))
+	finalPlatformVersion := "nomad"
 
-	tb.Detail("Locking app to prevent changes during the migration")
-	err = m.lockAppForMigration(ctx)
-	if err != nil {
-		return err
+	fmt.Printf("Recovery info: %v\n", m.recovery)
+
+	if m.recovery.machinesCreated {
+		// TODO: Unsure how to recover here.
```

Comment:
> If you already have machines, we can just scale those down and bring nomad back up, right?
this is from slack. would you want to implement it before we merge this?
or that's a different convo?